src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -6,10 +6,38 @@
 from sentry import features
 from sentry.api.bases.organization import OrganizationEndpoint
 from sentry.api.exceptions import ResourceDoesNotExist
-from sentry.api.paginator import OffsetPaginator
-from sentry.api.serializers import serialize
+from sentry.api.paginator import OffsetPaginator, CombinedQuerysetPaginator
+from sentry.api.serializers import serialize, CombinedRuleSerializer
 from sentry.incidents.models import AlertRule
 from sentry.incidents.endpoints.serializers import AlertRuleSerializer
+from sentry.snuba.dataset import Dataset
+from sentry.models import Rule, RuleStatus
+
+
+class OrganizationCombinedRuleIndexEndpoint(OrganizationEndpoint):
+    def get(self, request, organization):
+        """
+        Fetches alert rules and legacy rules for an organization
+        """
+        project_ids = self.get_requested_project_ids(request) or None
+
+        alert_rules = AlertRule.objects.fetch_for_organization(organization, project_ids)
+        if not features.has("organizations:performance-view", organization):
+            # Filter to only error alert rules
+            alert_rules = alert_rules.filter(snuba_query__dataset=Dataset.Events.value)
+
+        issue_rules = Rule.objects.filter(status__in=[RuleStatus.ACTIVE, RuleStatus.INACTIVE])
```

Comment:
Need to filter to all projects in org - else this is return all rules for all orgs.